cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -329,11 +368,11 @@ end
 # Policy
 ###############################################################################
 
-policy "policy_cheaper_regions" do
+policy "pol_cheaper_regions" do
   validate_each $ds_new_bc_cost_name do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows with cheaper data"
```

Comment:
The pattern we are updating policies to is the following:

`summary_template "{{ with index data 0 }}{{ .policy_name }}{{ end }}: {{ len data }} Cheaper Regions Found"`

policy_name is a field that should be added to the final datasource. You can obtain the policy_name with a ds_applied_policy datasource. Look at the "AWS Unused Volumes" policy if you want an example to use as a reference.